app/Legacy/V1/Controllers/Administration/SettingsController.php

```diff
@@ -403,6 +404,9 @@ public function getAll(GetSetAllSettingsRequest $request): Collection
 		return Configs::query()
 			->orderBy('cat')
 			->orderBy('id')
+			// Only display settings which are not part of SE
```

Comment:
Do we get those separately? Or should this be behind a gate? Or are we forcing these to be done the "normal" way?

Reply:
When using the Legacy front-end, you would be able to edit the SE settings without going through the middleware support check due to the way the code is implemented in the Legacy side.
As a result, someone would be able to change parameters that they are not supposed to.

In order to solve this, we do two things:
1. we do not display those settings. 
2. As you saw below, we also remove those parameters from the POST request if they were supplied. Effectively making those untouchable.

The idea being that if you want to modify those parameters (for things which are only available on the Vue front-end), you have to go through the new front-end.
The work around otherwise would be to disable v6, edit settings as per preference and re enable v6... Not what we want. :)

Reply:
Makes sense